roles-and-groups.md

```diff
@@ -0,0 +1,153 @@
+# Roles and Groups
+
+This file contains an overview of our rules for roles and groups.
+
+## Definition of general roles
+
+The Hiero project (and the LFDT) defines 3 different types of **roles**: **Contributors**, **Committers**, and **Maintainers**.
+Each person who contributes to Hiero must follow our [**Code of Conduct**](https://github.com/hiero-ledger/.github/blob/main/CODE_OF_CONDUCT.md).
+Those roles are not assigned globally for the Hiero project but per **project**.
+In most cases a project is a single GitHub repository, but it can also be a group of repositories that are related to each other.
+A person can have multiple roles in multiple projects.
+
+### Contributors
+
+Any person who contributes to a project is a Contributor. A contribution can be anything from opening an issue, to writing code, to writing documentation, etc.
+
+  - If a pull request has not received an approval within 2 business days or a discussion has no activity for more than 2 business days, the owner must reach out to the maintainers via our [hiero-general Discord channel](https://discord.lfdecentralizedtrust.org/).
+  - If a pull request or discussion has gone stale for more than 2 weeks, the repo maintainers may close them due to inactivity.
+
+We honor legitimate contributions to the project, and we want to make sure that all contributions are recognized.
+
+A Contributor can be nominated by a Committer or a Maintainer to become a Committer:
+
+  - **Start Contributing**: Submit pull requests, report bugs, improve documentation, review code. Show consistency and quality in your contributions.
+  - **Build Trust and Track Record**: Active committers will notice your contributions, your understanding, and responsiveness.
+  - **Get noticed**: Consistent and valuable contributions will demonstrate more project involvement and reliability.
+  - **Review eligibility**: A minimum number of approved PRs and a reasonable time contributing to the project (often 3-6 months) is the general acceptable baseline for eligibility.
+
+### Committers
+
+A Committer has specific rights for the project.
+
+As an example, a Committer can assign people to issues, merge PRs, or create branches directly in a repository of the project.
+Any Committer or Maintainer of the given project can nominate a Contributor to become a Committer based on the project's needs and following these steps:
+
+- **Define your project's needs**: A repository should have minimum 3 committers to ensure a project's sustainability and resilience. It will also allow for fairness in the votes. Project owners can choose to have more committers depending on the size of their project.
+- **Evaluate your project's performance**: Committers and Maintainers should monitor their project health and velocity and make decisions about whether the project needs more Committers.
```

Comment:
How should / can that be monitored

Reply:
This can be done via GH Insights, LFX Insights, Bitergia and overall performance of their releases. I think Mentioning the tools here is a little bit too much information but metrics in general  is something that Diane and I are working on and eventually get into a 5 or 10 min segment in the TSC.

In general, is up to Maintainers to manage their projects and making sure they are performing and keeping up with the demand. 

They are also welcomed to use any other metrics tool, the ones I mentioned here are a suggestion.